### DISCUSSION TITLE
你的照片DragGan了嗎?即將取代Photoshop成為修圖的新代名詞!

### DISCUSSION THREAD

**Discussion by BeaGle0928** [1101-GS4719-C/Python-Community-GS4719-C#1605] [General]
DragGan是一個基於深度學習的圖像編輯工具，全名為 Drag and Drop Generative Adversarial Network，意即托放式生成對抗網路。不同於Photoshop複雜的操作介面，DragGan 只需使用滑鼠設定關鍵點位置即可進行修圖。什麼是關鍵點?關鍵點是指圖像中能夠表示人或物體特徵的一些點，如:鼻子、嘴巴等。DragGan的原理簡單來說是透過不斷生成新的圖片使關鍵點逐漸靠近使用者設定的位子。
我認為DragGan最大的優勢在於它能夠改變圖中人、物的動作、方向等，這都是現今的修圖軟體無法做到的事。想像今天難得出國玩拍了一張照片，但照片中的自己卻不是自己喜歡的樣子，不論是眼神、笑容還是姿勢都不到位。這個時候DragGan就能幫你解決問題!
說到這邊想必大家都躍躍欲試了吧!
https://github.com/XingangPan/DragGAN
不知道大家會想要把DragGan用在哪呢?

**Comment by Fion-Lin** [comment]
感覺這個可以變成自動生成影片，雖然現在線上已經有一些免費的AI生成影片的軟體，但是他們有些是去定位一張照片的某些特徵點，套上一個移動的人偶，再去謀合到對應的位置使其進行移動，看起來就有點假假的，而且很不真實，動作也很不連貫。但我覺得透過DragGan應該可以解決這個問題，讓人分不出來是真是假。
